testing/btest/Baseline/spicy.tcp-eod-behavior/analyzer_debug-1024-no-fins.log

```diff
@@ -0,0 +1,2 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+SPICY_FOO	violation	expected 1024 bytes (136 available) (<...>/test.spicy:12:5-12:23)
```

Comment:
Iirc, this is actually not desired: #4015 changed semantics to not report an error if there's no regular shutdown.